source/blockchain_details.rst

```diff
@@ -0,0 +1,431 @@
+.. _blockchain_details:
+
+
+Core blockchain details
+^^^^^^^^^^^^^^^^^^^^^^^
+
+As any blockchain - the Credits blockchain consists of Blocks. And every
+block contains transactions that happened since previous block created.
+
+Unlike other blockchains the Credits blockchain also contains States,
+which are snapshots of the contents of the blockchain created after each
+block.
```

Comment:
Yes, it is technically "applying next block to the state", but I don't want to throw this at people in the very beginning. In here it doesn't bring any value afaics, but obscures meaning for an unprepared reader. I'd like to not use context specific terms before it is defined, and `application` in Credits Core context is defined and discussed much later in the text.

I've fixed English this way: 
> are snapshots of the contents of the blockchain created after each block is appended to the blockchain.